templates/project-ts/package.json

```diff
@@ -45,5 +43,8 @@
     "prettier": "^2.3.2",
     "ts-jest": "^27.0.7",
     "typescript": "^4.7.2"
+  },
+  "peerDependencies": {
+    "snarkyjs": "^0.4.1"
```

Comment:
Needs a bump to 0.4.3 now
https://github.com/o1-labs/snarkyjs/blob/main/package.json#L4

Reply:
0.4.4 now! we're moving fast 😁